lib/src/cli/command/bump_sub_command.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:cider/src/cli/console.dart';
+import 'package:cider/src/project.dart';
+import 'package:cider/src/cli/command/cider_command.dart';
+import 'package:version_manipulation/mutations.dart';
+
+class BumpSubCommand extends CiderCommand {
+  BumpSubCommand(
+    this.name, {
+    required this.description,
+    required this.mutation,
+    required Console printer,
```

Comment:
Unrelated to this PR, but we should rename this to `console`.

Reply:
Good catch! I realized that, too. I'll make another PR tomorrow for that then.